### ISSUE TITLE
mpdas as daemon

### ISSUE THREAD

**Issue by mistepien** [hrkfdn/mpdas#36]
it would be great to enjoy mpdas as daemon sending messages to syslog.

**Comment by TheReverend403** [comment]
https://aur.archlinux.org/cgit/aur.git/tree/mpdas.service?h=mpdas

**Comment by mistepien** [comment]
On 11/09/16 03:07, Lee Watson wrote:
> https://aur.archlinux.org/cgit/aur.git/tree/mpdas.service?h=mpdas
> 
Ok, są even busybox can do it.
But what about logging which would be obvious in real daemon mode?

**Comment by hrkfdn** [comment]
mpdas does daemonize when the config var or -d is set. is this specifically about sending messages to syslog?

**Comment by mistepien** [comment]
Yes, in embedded system you may have syslog in busybox and this will not save logs.
